buildkite/bazelci.py

```diff
@@ -1605,7 +1605,13 @@ def test_logs_for_status(bep_file, status):
 
     pos = 0
     while pos < len(raw_data):
-        bep_obj, size = decoder.raw_decode(raw_data[pos:])
+        bep_obj = None
```

Comment:
We can skip the initialization here, because in Python it's OK to initialize a variable in a try-catch block and use it afterwards.